examples/mechanics/crack_branching.cpp

```diff
@@ -90,13 +90,14 @@ void crackBranchingExample( const std::string filename )
     double dy = particles->dx[1];
     double b0 = sigma0 / dy;
 
-    CabanaPD::RegionBoundary plane1( low_corner[0], high_corner[0],
-                                     low_corner[1] - dy, low_corner[1] + dy,
-                                     low_corner[2], high_corner[2] );
-    CabanaPD::RegionBoundary plane2( low_corner[0], high_corner[0],
-                                     high_corner[1] - dy, high_corner[1] + dy,
-                                     low_corner[2], high_corner[2] );
-    std::vector<CabanaPD::RegionBoundary> planes = { plane1, plane2 };
+    CabanaPD::RegionBoundary<CabanaPD::Rectilinear> plane1(
```

Comment:
Why was crack branching changed for the fragmenting cylinder example?

Reply:
See changes in the BC file - there was previous no cylindrical option, so the naming/setup needed to be changed. That can be moved to a new PR, but only if necessary